jupyter_server/services/events/handlers.py

```diff
@@ -0,0 +1,79 @@
+"""A Websocket Handler for emitting Jupyter server events.
+
+.. versionadded:: 2.0
+"""
+import logging
+
+from jupyter_telemetry.eventlog import _skip_message
+from pythonjsonlogger import jsonlogger
+from tornado import web, websocket
+
+from jupyter_server.base.handlers import JupyterHandler
+
+AUTH_RESOURCE = "events"
+
+
+class TornadoWebSocketLoggingHandler(logging.Handler):
```

Comment:
Is there anything Tornado specific here? Is WebSocketLoggingHandler a better name to use here?

Reply:
Technically, yes; the `websocket` object and its `.write_message()` method is not a general API for websockets in Python. That said, I doubt anyone is using anything else by Tornado here, so the distinction probably isn't necessary.

Reply:
Renamed to `WebSocketLoggingHandler`.